### DISCUSSION TITLE
Dashboard for API User Monitoring and Configuration

### DISCUSSION THREAD

**Discussion by meteoDaniel** [fastapi/fastapi#5829] [Questions]
Dear FastAPI Community,

I am looking for a software that is able to generate JWT for users to retrieve data via the API. 

It should provide a dashboard where I can:

- Create a New User with an Access Token
- Define a usage Plan with Quota limits 
- Get notified when Quota Limits reached
- Visualisation of API use

Actually I am not able to find such a tool for a year , now.   But I am sure that there is one.  

Thanks a lot in advance ! 🙏

**Comment by anthonycorletti** [comment]
👋 Hey @meteoDaniel! I'd suggest using some simple middleware with FastAPI that counts authenticated requests and set limits however you like from there. Maybe this plugin could help: https://github.com/code-specialist/fastapi-auth-middleware. Hope this helps!
